### ISSUE TITLE
Support SimpleAggregateFunction column type

### ISSUE THREAD

**Issue by chelsEg** [lizhichao/one-ck#2]
Hello.

I have a table:

```
CREATE TABLE IF NOT EXISTS table_name_replica on cluster '{cluster}'
(
    uuid         UUID,
    entity_id    UInt32,
    parameter_id UInt64,
    creation_ts  SimpleAggregateFunction(max, DateTime64) DEFAULT NOW(),
    number       SimpleAggregateFunction(sum, UInt64) DEFAULT 1
) engine = ReplicatedAggregatingMergeTree(
    '/clickhouse/{installation}/{cluster}/tables/{shard}/{database}/table_name_replica',
    '{replica}'
)
PARTITION BY toYYYYMM(creation_ts)
ORDER BY (uuid, entity_id, parameter_id)
```

When I run query:

`$ck->query('SELECT * FROM table_name_replica');`

I have an error:

```
OneCk\CkException:
not supported type :simpleaggregatefunction(sum, uint64)

  at /application/vendor/lizhichao/one-ck/src/Types.php:546
  at OneCk\Types->decode('simpleaggregatefunction(sum, uint64)', 2)
     (/application/vendor/lizhichao/one-ck/src/Types.php:604)
  at OneCk\Types->unpack('simpleaggregatefunction(sum, uint64)', 2)
     (/application/vendor/lizhichao/one-ck/src/Client.php:182)
  at OneCk\Client->readData()
     (/application/vendor/lizhichao/one-ck/src/Client.php:111)
  at OneCk\Client->receive()
     (/application/vendor/lizhichao/one-ck/src/Client.php:344)
  at OneCk\Client->writeEnd()
     (/application/vendor/lizhichao/one-ck/src/Client.php:259)
  at OneCk\Client->query('SELECT * FROM table_name_replica')
```

Can you provide support Clickhouse [SimpleAggregateFunctions](https://clickhouse.tech/docs/en/sql-reference/data-types/simpleaggregatefunction/)?

**Comment by lizhichao** [comment]
Nice job 😊

**Comment by chelsEg** [comment]
> Nice job 😊

How do you release this fix?

**Comment by lizhichao** [comment]
> > Nice job 😊
> 
> When do you release this fix?

Released at 0.1.12
